src/core/errors.tsx

```diff
@@ -1,11 +1,11 @@
 import { isFetchError } from '@grafana/runtime';
-import { Alert } from '@grafana/ui';
+import { Alert, AlertVariant } from '@grafana/ui';
 import { errorCodes } from '../datasources/data-frame/constants';
 import React, { useState, useEffect } from 'react';
 import { useTimeoutFn } from 'react-use';
 import { isSystemLinkError } from './utils';
 
-export const FloatingError = ({ message = '' }) => {
+export const FloatingError = ({ message = '', innerMessage = '', severity = 'error' }) => {
```

Comment:
```suggestion
export const FloatingError = ({ message = '', innerMessage = '', severity: AlertVariant = 'error' }) => {
```

nit: The proper typing would disallow other values to be passed from the caller and avoid the type conversion in the `Alert.severity` prop

Reply:
Have added a type prop to make it type strict